### PR TITLE
remove old migration

### PR DESCRIPTION
**Pull request by xlc**

### CODE REVIEW

File: runtime/karura/src/lib.rs [hunk 1]
```diff
@@ -2188,7 +2188,7 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsWithSystem,
-	(TransactionPaymentMigration, XcmInterfaceMigrationV1),
+	XcmInterfaceMigrationV1,
```
Comment (zqhxuyuan) [inline runtime/karura/src/lib.rs:2191]: XcmInterfaceMigrationV1 not applied?

Comment (xlc) [reply runtime/karura/src/lib.rs:2191]: Not yet

Comment (xlc) [reply runtime/karura/src/lib.rs:2191]: https://github.com/AcalaNetwork/Acala/pull/1893

**Comment by zqhxuyuan** [comment]
LGTM.
For some note, the origin is wrong in TransactionPaymentMigration(should be root), so the migration is taken by manual council dispatch call and validated ok:
https://karura.subsquare.io/council/motion/218
https://karura.subsquare.io/council/motion/219

**Pull request merged**